lib/modules/collapseWhitespace.es6

```diff
@@ -8,11 +7,15 @@ const noWhitespaceCollapseElements = new Set([
     'textarea'
 ]);
 
+const indentPattern = /[\f\n\r\t\v]{1,}/g;
+const whitespacePattern = /[\f\n\r\t\v ]{1,}/g;
+const NONE = '';
+const SINGLE_SPACE = ' ';
+const validOptions = ['all', 'aggressive', 'conservative'];
+
 /** Collapses redundant whitespaces */
 export default function collapseWhitespace(tree, options, collapseType, tag) {
-    if (collapseType !== 'all') {
-        collapseType = 'conservative';
-    }
+    collapseType = (collapseType && validOptions.some(o=>o === collapseType)) ? collapseType : 'conservative';
```

Comment:
This could be simplified to
```suggestion
    collapseType = validOptions.includes(collapseType) ? collapseType : 'conservative';
```

Reply:
Oh boy, I didn't know about the .includes method, I learnt something awesome today, thanks!

Reply:
> so do you want a new PR for this simplification or do you want to just let it go?

It's up to you :-)

Reply:
Always better to keep things tidy imho : https://github.com/posthtml/htmlnano/pull/91